### ISSUE TITLE
Undefined Index reportId

### ISSUE THREAD

**Issue by pivnicek** [keboola/sklik-extractor#16]
```
ErrorException:Undefined index: reportId {\"errFile\":\"\/code\/src\/Extractor.php\",\"errLine\":76,\"errCode\":0,\"errTrace\":\"#0 \/code\/src\/Extractor.php(76): Keboola\\\\Component\\\\BaseComponent::Keboola\\\\Component\\\\{closure}(8, 'Undefined index...', '\/code\/src\/Extra...', 76, Array)\\n#1 \/code\/src\/Component.php(26): Keboola\\\\SklikExtractor\\\\Extractor->run(Object(Keboola\\\\SklikExtractor\\\\Config))\\n#2 \/code\/src\/run.php(14): Keboola\\\\SklikExtractor\\\\Component->run()\\n#3 {main}\",\"errPrevious\":\"\"}
```

**Comment by JakubMatejka** [comment]
@pivnicek Could you please find out on which resource this happened?

**Comment by pivnicek** [comment]
@JakubMatejka looks like it was `campaigns`

**Comment by JakubMatejka** [comment]
That's weird because according to the docs it should return the `reportId` (http://api.sklik.cz/drak/campaigns.createReport.html). So I created a PR which should log the params and response and end with user error: https://github.com/keboola/sklik-extractor/pull/17

**Comment by JakubMatejka** [comment]
Ok, released as 3.0.15 (https://github.com/keboola/sklik-extractor/releases/tag/3.0.15). So, let's see how the configuration ends now.
